sdk/identity/identity/package.json

```diff
@@ -10,12 +10,12 @@
   "//sampleConfiguration": {
     "productName": "Azure Identity",
     "productSlugs": [
-      "azure",
-      "azure-active-directory"
+      "entra",
```

Comment:
do we need to re-run sample generation to pull in these updates?

Reply:
Ooh maybe. I can try running it before doing the bug fix release